### PR TITLE
Add first pass at behavior rig

### PR DESCRIPTION
**Pull request by saskiad**
added a simple behavior rig schema, and made a behavior folder

### CODE REVIEW

**Comment by saskiad** [comment]
Something we need to think about is whether the behavior "rig" is something that gets inherited into ephys and ophys rigs? Or if it should just be a parallel for a behavior only context. We can discuss later - I think that's beyond the scope of this PR

**Comment by saskiad** [comment]
closes #284

File: src/aind_data_schema/behavior/behavior_rig.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    HarpDevice,
+    Monitor,
+    Treadmill,
+    Tube,
+)
+
+class BehaviorRig(AindCoreModel):
+    """Description of an behavior rig"""
+
+    describedBy: str = Field(
```
Comment (dyf) [inline src/aind_data_schema/behavior/behavior_rig.py:24]: you can remove `describedBy`.  It's automatically included by inheriting from `AindCoreModel`.

File: src/aind_data_schema/behavior/behavior_session.py [hunk 2]
```diff
@@ -7,13 +7,19 @@
...
-from .base import AindCoreModel
-from .procedures import VolumeUnit, WeightUnit
+from ..base import AindCoreModel
+from ..procedures import VolumeUnit, WeightUnit
 
 
 class BehaviorSession(AindCoreModel):
     """Description of a behavior session"""
 
+    describedBy: str = Field(
```
Comment (dyf) [inline src/aind_data_schema/behavior/behavior_session.py:17]: ditto

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/aind_data_schema/behavior/behavior_rig.py (hunk 1 above)
Comment (saskiad) [reply src/aind_data_schema/behavior/behavior_rig.py]: Ah! excellent. Let's remove these from other schemas - I'll make an issue

**Review by dyf — APPROVED** [review]

**Pull request merged**